views/item.py

```diff
@@ -0,0 +1,73 @@
+from flask_restful import Resource, reqparse
+from http.client import CREATED
+from http.client import NO_CONTENT
+from http.client import NOT_FOUND
+from http.client import OK
+from http.client import BAD_REQUEST
+import uuid
+import models
+
+
+class Items(Resource):
+    def get(self):
+        return [obj.json() for obj in models.Item.select()], OK
+
+    def post(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('name', type=str, required=True)
+        parser.add_argument('price', type=int, required=True)
+        parser.add_argument('description', type=str, required=True)
+        try:
+            args = parser.parse_args()
+        except ValueError:
+            return None, BAD_REQUEST
```

Comment:
Mi sembra che non ce ne sia bisogno, perché la parse_args già fa un abort con BAD_REQUEST se fallisce. Sbaglio?
In aggiunta, non avendo messo il parametro `strict=True` è possibile passargli anche altri parametri..